packages/swc-plugin/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@swc-contrib/next-intl-plugin",
```

Comment:
Would it be ok for you if we publish this under a `next-intl` scope? Mostly for me to have publish access.

If you prefer, we can also move it to https://github.com/swc-project/plugins though, there might just be a practical advantage of keeping it in this repo due to making it easier to make changes across both the SWC plugin and the loader implementation.

It's your call though, either option is fine by me!